docs/api-guide/validators.md

```diff
@@ -290,13 +290,18 @@ To write a class-based validator, use the `__call__` method. Class-based validat
                 message = 'This field must be a multiple of %d.' % self.base
                 raise serializers.ValidationError(message)
 
-#### Using `set_context()`
+#### Accessing the context
 
-In some advanced cases you might want a validator to be passed the serializer field it is being used with as additional context. You can do so by declaring a `set_context` method on a class-based validator.
+In some advanced cases you might want a validator to be passed the serializer
+field it is being used with as additional context. You can do so by using
+`rest_framework.validators.ContextBasedValidator` as a base class for the
+validator. The `__call__` method will then be called with the `serializer_field`
+or `serializer` as an additional argument.
```

Comment:
`ContextBasedValidator` is potentially confusing, as the serializer `context` is an entirely different concept. Possibly `FieldContextValidator` or `FieldBasedValidator`?